jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt

```diff
@@ -372,6 +376,84 @@ class CodeModernizerManager(private val project: Project) : PersistentStateCompo
         managerState.flags[StateFlags.IS_ONGOING] = false
     }
 
+    /**
+    helper function to generate 1p dependency metrics
+    makes GET requests to maven central to check for 1p dependencies for a small subset of the project's
+    dependencies
+     */
+    private fun makeGETRequests(groupId: String, artifactId: String, version: String, jobId: JobId): Boolean {
+        try{
+            val response = getTextFromUrl("https://search.maven.org/solrsearch/select?q=g:$groupId%20AND%20a:$artifactId%20AND%20v:$version%20AND%20p:jar&rows=20&wt=json")
+                try {
+                    val json: JsonObject = Json.parseToJsonElement(response) as JsonObject;
+                    val responses = json["response"] as? JsonObject;
+                    val numFound = responses?.get("numFound");
+
+
+                    if(numFound.toString() == "0"){
```

Comment:
letting the deserializer coerce the response into a data class will allow you to do an int comparison on an int instead of a string comparison

Reply:
if you're intentionally interacting with raw `JsonElement`s, you generally want to coerce the JsonElement into the proper type instead of just doing `toString()`